react_native_fundamentals/presentation.html

```diff
@@ -2,43 +2,1027 @@
+          </aside>
+        </section>
+
+        <section class="bullets-and-picture-slide">
+          <h4 class="slide-name">Ionic</h4>
+          <div>
+            <img src="assets/images/ionic_big.png" alt="simple" />
+            <div class="pros-cons-section">
+              <p class="big">
+                Pros:
+              </p>
+              <ul>
+                <li>Short development cycle</li>
+                <li>Easy-to-learn</li>
+              </ul>
+              <p class="big">
+                Cons:
+              </p>
+              <ul>
+                <li>Lower performance</li>
+                <li>App will look the same regardless of platform</li>
+              </ul>
+            </div>
+          </div>
+          <aside class="notes">
+            An open source SDK launched in 2013. It uses multiple technologies (including HTML5, CSS, and JS) to develop and run applications and access the <strong>Cordova wrapper</strong> to use native platform controllers. Ionic is a successor of Cordova, but with improved documentation and powerful command line interface. Another technology that influenced the framework is Angular. Hence, if you have a background of the mentioned tools, you’ll have no trouble with creating apps using Ionic.<br><strong>Advantages of Ionic.</strong><br>With Ionic, you do not have to spend much time working with native languages or other platform-specific issues. There is a plethora of native-styled UI components. As a result, the development cycle is shorter. The framework is easy-to-learn, and the bulk of engineers are familiar with the technologies under the Ionic’s hood. Fast prototyping with Ionic is possible due to the preset templates not to mention a variety of third-party starter kits.<br><strong>Disadvantages of Ionic.</strong><br>Performance is the price you have to pay for the advantages of hybrid app development. Performance issues are possible in case of numerous callbacks sent to the native code. Also, native UI look is what you should forget about when using Ionic. Your app will look the same regardless of the platform/device it runs on.<br>
```

Comment:
I feel like it's too much text for these speaker notes.

Reply:
Done